validate_sptx/examples/experiment/codebook.json

```diff
@@ -0,0 +1,24 @@
+[
```

Comment:
I don't understand the difference between this and validate_sptx/examples/codebook/codebook.json.

Reply:
None, but I assume tests should refer only to files in a given directory (to prevent unintended breakages). Happy with any of:

 * re-use from other directories
 * symlink to other directories
 * copy but use more explicit naming